src/frontend/Komponenter/Behandling/Totrinnskontroll/FatterVedtak.tsx

```diff
@@ -69,6 +77,24 @@ const FatterVedtak: React.FC<{
             (begrunnelse || '').length > 0 &&
             årsakerUnderkjent.length > 0);
 
+    const hentSimuleringStatus = useCallback(
+        (behandlingId: string) => {
+            axiosRequest<boolean, null>({
+                method: 'GET',
+                url: `/familie-ef-sak/api/simulering/resultatstatus/${behandlingId}`,
+            }).then((res: Ressurs<boolean>) => settSimuleringStatusFeil(res.data));
+        },
+        [axiosRequest]
+    );
+
+    useEffect(() => {
+        if (behandling.steg === Steg.BESLUTTE_VEDTAK) {
+            hentSimuleringStatus(behandling.id);
+        } else {
+            settSimuleringStatusFeil(false);
+        }
```

Comment:
```suggestion
        if (behandling.steg === Steg.BESLUTTE_VEDTAK) {
            hentSimuleringStatus(behandling.id);
        }
```

Jeg ser nå at vi ikke trenger å sette statusen til false, siden den blir initialisert med false :)